core/src/main/java/org/elasticsearch/transport/netty/NettyTransport.java

```diff
@@ -225,7 +243,8 @@
     final ScheduledPing scheduledPing;
 
     @Inject
-    public NettyTransport(Settings settings, ThreadPool threadPool, NetworkService networkService, BigArrays bigArrays, Version version, NamedWriteableRegistry namedWriteableRegistry) {
+    public NettyTransport(Settings settings, ThreadPool threadPool, NetworkService networkService, BigArrays bigArrays, Version version,
+            NamedWriteableRegistry namedWriteableRegistry) {
```

Comment:
Its just what Eclipse seems to like. I've not tried to fight it. If we want to change the defaults I'm happy to put instructions in CONTRIBUTING.md for it.